README.md

```diff
@@ -2,13 +2,13 @@
 
 # Planck.js<sup>&alpha;</sup>
 
-Planck.js is JavaScript rewrite of Box2D physics engine for cross-platform HTML5 game development. **[Check out demos!](http://piqnt.com/planck.js/)**
+Planck.js is JavaScript rewrite of Box2D physics engine for cross-platform HTML5 game development. **[Check out some demos!](http://piqnt.com/planck.js/)**
```

Comment:
It's a matter of formality and style, if that is what you want then you can keep it that way